<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-HB25A9N" failures="2" tests="29" name="Arithmetic Function Unit Test" time="0.073" errors="0" timestamp="2024-02-28T21:56:22 MSK">
  <testcase classname="org.rntgroup.CalcUnitTests.SumLongTest" name="unitSumLongTest" time="0.0"/>
  <testcase classname="org.rntgroup.CalcUnitTests.SumLongTest" name="unitSumLongTest" time="0.001"/>
  <testcase classname="org.rntgroup.CalcUnitTests.SumLongTest" name="unitSumLongTest" time="0.0"/>
  <testcase classname="org.rntgroup.CalcUnitTests.SumLongTest" name="unitSumLongTest" time="0.0"/>
  <testcase classname="org.rntgroup.CalcUnitTests.SumLongTest" name="unitSumLongTest" time="0.0"/>
  <testcase classname="org.rntgroup.CalcUnitTests.SumDoubleTest" name="unitSumDoubleTest" time="0.0"/>
  <testcase classname="org.rntgroup.CalcUnitTests.SumDoubleTest" name="unitSumDoubleTest" time="0.001"/>
  <testcase classname="org.rntgroup.CalcUnitTests.SumDoubleTest" name="unitSumDoubleTest" time="0.0"/>
  <testcase classname="org.rntgroup.CalcUnitTests.SumDoubleTest" name="unitSumDoubleTest" time="0.001"/>
  <testcase classname="org.rntgroup.CalcUnitTests.SumDoubleTest" name="unitSumDoubleTest" time="0.0"/>
  <testcase classname="org.rntgroup.CalcUnitTests.SumDoubleTest" name="unitSumDoubleTest" time="0.0"/>
  <testcase classname="org.rntgroup.CalcUnitTests.SumDoubleTest" name="unitSumDoubleTest" time="0.0"/>
  <testcase classname="org.rntgroup.CalcUnitTests.SumDoubleTest" name="unitSumDoubleTest" time="0.001"/>
  <testcase classname="org.rntgroup.CalcUnitTests.SubDoubleTest" name="unitSubDoubleTest" time="0.0"/>
  <testcase classname="org.rntgroup.CalcUnitTests.SubDoubleTest" name="unitSubDoubleTest" time="0.0"/>
  <testcase classname="org.rntgroup.CalcUnitTests.SubDoubleTest" name="unitSubDoubleTest" time="0.0"/>
  <testcase classname="org.rntgroup.CalcUnitTests.SubDoubleTest" name="unitSubDoubleTest" time="0.0"/>
  <testcase classname="org.rntgroup.CalcUnitTests.SubDoubleTest" name="unitSubDoubleTest" time="0.0"/>
  <testcase classname="org.rntgroup.CalcUnitTests.SubDoubleTest" name="unitSubDoubleTest" time="0.0"/>
  <testcase classname="org.rntgroup.CalcUnitTests.MultiplyDoubleTest" name="unitMultiplyDoubleTest" time="0.0"/>
  <testcase classname="org.rntgroup.CalcUnitTests.MultiplyDoubleTest" name="unitMultiplyDoubleTest" time="0.0"/>
  <testcase classname="org.rntgroup.CalcUnitTests.MultiplyDoubleTest" name="unitMultiplyDoubleTest" time="0.0"/>
  <testcase classname="org.rntgroup.CalcUnitTests.MultiplyDoubleTest" name="unitMultiplyDoubleTest" time="0.01">
    <failure type="java.lang.AssertionError" message="Result is not correct! expected [154.53] but found [154.0]">
      <![CDATA[java.lang.AssertionError: Result is not correct! expected [154.53] but found [154.0]
at org.rntgroup.CalcUnitTests.MultiplyDoubleTest.unitMultiplyDoubleTest(MultiplyDoubleTest.java:25)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- unitMultiplyDoubleTest -->
  <testcase classname="org.rntgroup.CalcUnitTests.MultiplyDoubleTest" name="unitMultiplyDoubleTest" time="0.0">
    <failure type="java.lang.AssertionError" message="Result is not correct! expected [133.32] but found [133.0]">
      <![CDATA[java.lang.AssertionError: Result is not correct! expected [133.32] but found [133.0]
at org.rntgroup.CalcUnitTests.MultiplyDoubleTest.unitMultiplyDoubleTest(MultiplyDoubleTest.java:25)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- unitMultiplyDoubleTest -->
  <testcase classname="org.rntgroup.CalcUnitTests.MultiplyLongTest" name="unitMultiplyLongTest" time="0.001"/>
  <testcase classname="org.rntgroup.CalcUnitTests.MultiplyLongTest" name="unitMultiplyLongTest" time="0.0"/>
  <testcase classname="org.rntgroup.CalcUnitTests.MultiplyLongTest" name="unitMultiplyLongTest" time="0.0"/>
  <testcase classname="org.rntgroup.CalcUnitTests.MultiplyLongTest" name="unitMultiplyLongTest" time="0.0"/>
  <testcase classname="org.rntgroup.CalcUnitTests.MultiplyLongTest" name="unitMultiplyLongTest" time="0.0"/>
</testsuite> <!-- Arithmetic Function Unit Test -->
